cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/explore/01/discover_traces.spec.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DATASOURCE_NAME } from '../../../../../../utils/apps/explore/constants';
+import { getRandomizedWorkspaceName } from '../../../../../../utils/apps/explore/shared';
+import { prepareTestSuite } from '../../../../../../utils/helpers';
+import { INDEX_WITH_TIME_1 } from '../../../../../../utils/constants';
+
+const workspaceName = getRandomizedWorkspaceName();
+const INDEX_PATTERN_NAME = 'otel-v1-apm-span-*';
+const TIME_FIELD_NAME = 'endTime';
+
+const traceTestSuite = () => {
+  describe('Trace Exploration Tests', () => {
+    before(() => {
+      cy.osd.setupWorkspaceAndDataSourceWithIndices(workspaceName, [INDEX_WITH_TIME_1]);
+    });
+
+    after(() => {
+      cy.osd.cleanupWorkspaceAndDataSourceAndIndices(workspaceName, [INDEX_WITH_TIME_1]);
+    });
+
+    it('should show empty state when no index pattern exists', function () {
+      cy.osd.navigateToWorkSpaceSpecificPage({
+        workspaceName: workspaceName,
+        page: 'explore/traces',
+        isEnhancement: true,
+      });
+      cy.osd.waitForLoader(true);
+      cy.getElementByTestId('discoverNoIndexPatterns').should('be.visible');
+    });
+
+    it('should create trace index pattern and navigate to trace details', function () {
+      // Navigate to sample data page
+      cy.osd.navigateToWorkSpaceSpecificPage({
+        workspaceName: workspaceName,
+        page: 'import_sample_data',
```

Comment:
there's an api we can invoke to just install the sample data

Reply:
I didn't see an example used in our existing test and am hesitant to introduce the command to be used in neo as this use of sample data is most likely temporary

Reply:
we should have just ported over all the existing commands instead of creating new ones. 

you can copy these over https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress/utils/commands.js#L614-L646

Reply:
Tested and not working out of the box. Will look into this as follow-up if it sample data is maintained.